daemon/openrazer_daemon/hardware/keyboards.py

```diff
@@ -401,6 +401,23 @@ class RazerBlackWidowV3ProWired(_RippleKeyboard):
     DEVICE_IMAGE = "https://dl.razerzone.com/src/3809-1-EN-v1.png"
 
 
+class RazerBlackWidowV3ProWireless(RazerBlackWidowV3ProWired):
+    """
+    Class for the Razer BlackWidow v3 Pro 2.4 Ghz Wireless
+    """
+
+    USB_PID = 0x025C
+    METHODS = ['get_device_type_keyboard', 'set_wave_effect', 'set_static_effect', 'set_spectrum_effect',
+               'set_reactive_effect', 'set_none_effect', 'set_breath_random_effect', 'set_breath_single_effect', 'set_breath_dual_effect',
+               'set_custom_effect', 'set_key_row', 'get_game_mode', 'set_game_mode', 'get_macro_mode', 'set_macro_mode',
+               'get_macro_effect', 'set_macro_effect', 'get_macros', 'delete_macro', 'add_macro',
+               'set_starlight_random_effect', 'set_starlight_single_effect', 'set_starlight_dual_effect',
+               'set_ripple_effect', 'set_ripple_effect_random_colour',
```

Comment:
So as commented in the previous PR, I've added it and:
 - `ripple` does not work on wireless OR when I plug the keyboard
 - same for `reactive` effect, I see nothing happening in both cases

Reply:
So you're saying both ripple and reactive effects also don't work in Wired mode? If it doesn't maybe we should remove it from both Wireless and Wired?

Reply:
That was at least my experience when I tested the latest PPA version available (3.7.0 I think), on my laptop (ubuntu 23.10). All other effects were doing something, except those two. I am unsure whether it is supposed to be working or not, though.

Reply:
(also reproduced when using current driver and daemon from git master, with polychromatic-controller, the effect was selectable but nothing would happen and no error reported anywhere as much as I can tell)

Reply:
I'd say you should add a commit to remove those effects from the Wired version, and not add them in the Wireless version in the first place.

Reply:
So, look at the other reply, it seems to work.

Reply:
Or not, I'm a bit lost

Reply:
Ok, so Ripple works in wired mode, not in wireless for me.

Reply:
@z3ntu I am unsure if it's a bug driver-side or daemon side, let me know if you have ideas on what to test there?

Reply:
Could those warnings be related? `razerkbd: Command timed out. status: 04 transaction_id.id: 3f remaining_packets: 00 protocol_type: 00 data_size: 0c, command_class: 0f, command_id.id: 02 Params: 00000800000000000000000000000000 .`

Reply:
Yeah this message is definitely not good. You could try to increase the timeouts in the driver and see if that improves the situation.

Reply:
What should we do ? Maybe it would be better to just land the current PR without this feature and file a follow up issue ?

Reply:
Removing a broken effect is definitely an option, but I'd still like to see you try my previous comment, so increase timeouts in the driver, maybe that just makes it work.

Reply:
ok but which timeout? I dont see anything obvious in `razer_attr_write_matrix_effect_custom`

Reply:
Those?

```
+    case USB_DEVICE_ID_RAZER_BLACKWIDOW_V3_PRO_WIRELESS:
+        *report_index = 0x02;
+        *response_index = 0x02;
+        *wait_min = RAZER_BLACKWIDOW_V3_WIRELESS_WAIT_MIN_US;
+        *wait_max = RAZER_BLACKWIDOW_V3_WIRELESS_WAIT_MAX_US;
+        break;
```

Reply:
yes, wait_min / wait_max there. Update both values to be e.g. 10 times as high as they are right now

Reply:
Sorry for the delay, I'll try things

Reply:
Unfortunately, it does not help. I tried hacking a few things, but I dont really have much time to investigate more. Happy to test any suggestion though.

FTR, when I toggle "Ripple" in Polychromatic UI, not only it does not work on the keyboard, but the next effect I select is also not working ; I have to select again a different one before all effects (except Ripple) works.

Reply:
Maybe I've asked this before, but apart from ripple does everything work as expected? We can just get the device in without ripple then. If somebody wants this later they can try and get it working then but we don't need to hold up everything because of ripple.

Reply:
As much as I can remember, yes, only ripple was behaving erratically.

Reply:
Let me rebase and remove ripple for now